### ISSUE TITLE
Property 'byteLength' does not exist on type 'string'

### ISSUE THREAD

**Issue by gmfc** [colyseus/colyseus-loadtest#2]
Running colyseus-loadtest gives out this error:

```Bash
$ npx colyseus-loadtest --help                                                                 
⨯ Unable to compile TypeScript:
node_modules/@colyseus/loadtest/src/index.ts(276,31): error TS2339: Property 'byteLength'
does  not exist on type 'string | ArrayBuffer | SharedArrayBuffer | Blob | ArrayBufferView'.
Property 'byteLength' does not exist on type 'string'.
```
NPM Version: 6.4.1
NodeJS Version: v10.14.1

It references this line:
https://github.com/colyseus/colyseus-loadtest/blob/2cca4122494b6ccb0acaa581d6645c81198de326/src/index.ts#L276

**Comment by endel** [comment]
Oi @gmfc 👋 

That's odd, which TypeScript version are you using in your project? I couldn't reproduce using the latest `typescript@^3.3.3`.

Just released `@colyseus/loadtest@0.2.2` forcing this argument to be of `ArrayBuffer` type (https://github.com/colyseus/colyseus-loadtest/commit/50f8184336e858144afc14842b62196f9a0479a4)

Let me know if it works for you! Cheers

**Comment by gmfc** [comment]
Re-installed all the things, working like a charm, thanks!
